lib/served/serializers/json_api/error.rb

```diff
@@ -46,7 +46,7 @@ def source_pointer
 
         def source
           res = attrs.fetch(:source, {})
```

Comment:
Can the value at `:source` ever be falsy? If not, you could just skip the rest and directly return:
`attrs.fetch(:source, {})`